### PR TITLE
wrong exec on kustomize, upgraded yq and fixed kustomize command

### PR DESCRIPTION
**Pull request by brondum**

### CODE REVIEW

File: orb/orb.yml [hunk 1]
```diff
@@ -315,11 +315,11 @@ commands:
             KUBE_VERSION=$(kubectl version -o yaml | yq .clientVersion.minor)
```
Comment (Gaardsholt) [inline orb/orb.yml:315]: Should this line be like this?
```suggestion
            KUBE_VERSION=$(kubectl version -o yaml | yq eval .clientVersion.minor -)
```

File: orb/orb.yml [hunk 2]
```diff
@@ -300,7 +300,7 @@ commands:
             export DEPLOY_FILE=<< parameters.deploy-file >>
             export APP_NAME=<< parameters.app-name >>
             export SECRET_FILE=<< parameters.secret-file >>
-            export SECRETS=$(yq read $SECRET_FILE -j)
+            export SECRETS=$(yq -o=json $SECRET_FILE)
```
Comment (Gaardsholt) [inline orb/orb.yml:303]: Shouldn't this be like this?
```suggestion
            export SECRETS=$(yq eval -o json $SECRET_FILE)
```

File: orb/orb.yml [hunk 3]
```diff
@@ -257,7 +257,7 @@ commands:
       - run:
```
Comment (Gaardsholt) [inline orb/orb.yml:257]: Seen as we are downloading and installing yq in more than one place - should we extract this step and put it into an yaml anchor or a command?

File: orb/orb.yml (hunk 2 above)
Comment (brondum) [reply orb/orb.yml:303]: Why do you want the eval command ?

File: orb/orb.yml (hunk 1 above)
Comment (Gaardsholt) [reply orb/orb.yml]: And I guess it's the same with this one - we don't need to add the eval

File: orb/orb.yml (hunk 3 above)
Comment (brondum) [reply orb/orb.yml]: Check the latest commit, i am not sure the yaml is correct though

**Review by Gaardsholt — APPROVED** [review]

**Pull request merged**